hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/UnassignProcedure.java

```diff
@@ -80,9 +78,6 @@ protected void serializeStateData(ProcedureStateSerializer serializer) throws IO
     if (this.destinationServer != null) {
       state.setDestinationServer(ProtobufUtil.toServerName(destinationServer));
     }
-    if (force) {
-      state.setForce(true);
```

Comment:
we should add a comment to the protobuf definition for  `UnassignRegionStateData` in `MasterProcedure` that the field for force is ignored.

Reply:
Will fix later. Thanks.